rfcs/proposed/zip_view/README.md

```diff
@@ -0,0 +1,56 @@
+# zip_view Support for the oneDPL Range APIs with C++20
+
+## Introduction
+`std::ranges::zip_view` is a powerful utility that enables developers to combine two or more ranges into a single view,
+where each element is represented as a tuple containing corresponding elements from each input range.
+
+## Motivations
+`std::ranges::zip_view` is a convenient way to combine multiple ranges into a single view, where each element of
+the resulting range is a tuple containing one element from each of the input ranges. This can be particularly
+useful for iterating over multiple collections in parallel. `std::ranges::zip_view` was introduced in C++23,
+but many developers are still using C++20 standard. So, oneDPL introduces `oneapi::dpl::ranges::zip_view`,
+with the same API and functionality as `std::ranges::zip_view`.
+
+In case of C++23 `oneapi::dpl::ranges::zip_view` using also makes sense at least for the device policies, because
+`std::ranges::zip_view` C++23 still is not device copyable. Any wrapper over `std::tuple` C++23 is not device copyable. (https://godbolt.org/z/brfvcMeM6)
+There are another technical issues with `std::tuple` (see below for the details).
+So, oneDPL tuple-like concept  should be proposed into the oneDPL spec.
+
+### Key Requirements
+`oneapi::dpl::ranges::zip_view` should be:
+- compilable with C++20 version (minimum)
+- API-compliant with `std::ranges::zip_view`
+- in case of a device usage: a device copyable view if the all "underlying" views are device copyable views.
+- To provide a transitive device copyability oneDPL tuple-like type underhood is proposed - `oneapi::dpl::__internal::tuple`.
```

Comment:
I think which tuple to use is an implementation detail. What we really want is that our `zip_view` is trivially copyable if all the underlying views are - this is the way to have transitive device copyability for view pipelines.

Reply:
Yes, I think "to have transitive device copyability for view pipelines" - very strong argument to want that our zip_view is trivially copyable.
BTW, the proposed implementation has such property. The relevant test also was added.